k8sHelm/action.yml

```diff
@@ -0,0 +1,38 @@
+name: "Helm package, push & install"
+description: "Helm package, push & install"
+
+runs:
+  using: "composite"
+  steps:
+      - uses: google-github-actions/auth@v0
+        with:
+          credentials_json: '${{ secrets.GKE_PUSHER_CREDENTIALS }}'
```

Comment:
replace this secret with org level secret which I have created
secrets.GKE_SA_GHA_PUSHER_CREDENTIALS